Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -209,13 +209,12 @@ class Navigator {
 
     // MARK: - Navigator Updates
 
-    func setRoutes(_ routes: Routes?, uuid: UUID, completion: @escaping (Result<RouteInfo, Error>) -> Void) {
-        if let routes = routes {
-            routeCoordinator.beginActiveNavigation(with: routes, uuid: uuid, completion: completion)
-        }
-        else {
-            routeCoordinator.endActiveNavigation(with: uuid, completion: completion)
-        }
+    func setRoutes(_ routes: NavigationRoutes, uuid: UUID, legIndex: UInt32, completion: @escaping (Result<RouteInfo, Error>) -> Void) {
+        routeCoordinator.beginActiveNavigation(with: routes, uuid: uuid, legIndex: legIndex, completion: completion)
+    }
+    
+    func unsetRoutes(uuid: UUID, completion: @escaping (Result<RouteInfo, Error>) -> Void) {
+        routeCoordinator.endActiveNavigation(with: uuid, completion: completion)
```

Comment:
I broke out a separate method for unsetting the route, because more than one parameter has different expected values, and the implementation ends up being totally different.